distributions/standalone/src/main/resources/conf/wrapper.conf

```diff
@@ -39,6 +39,9 @@ wrapper.java.additional.15=-Dorg.quartz.scheduler.skipUpdateCheck=true
 # For large maps (>512) make the hash function depend on the JVM instance to prevent collision attacks (see MULE-11206)
 wrapper.java.additional.16=-Djdk.map.althashing.threshold=512
 
+# Set Default Grizzly Memory Manager.
+# wrapper.java.additional.17=-Dorg.glassfish.grizzly.DEFAULT_MEMORY_MANAGER=org.glassfish.grizzly.memory.HeapMemoryManager
```

Comment:
Do not set the number if it's commented, and change the explanation to "Allows modifying Grizzly's memory manager"